tests/Avalonia.RenderTests/Media/VisualBrushTests.cs

```diff
@@ -14,6 +14,8 @@ namespace Avalonia.Direct2D1.RenderTests.Media
 {
     public class VisualBrushTests : TestBase
     {
+        private static readonly string s_visualBrushText = "           ";
```

Comment:
Seems a little strange to be using text that consists just of spaces? Is this a hack until text rendering is fixed cross-platform? If so would be worth adding a comment explaining what's happening.

Reply:
This "hack" makes sure the measure stays the same. This works because noting is actually rendered. Will add a comment that this will most likely be fixed when all render backends use Freetype. Still, this does not guarantee that all render backends will produce the same results.

Reply:
I have added a comment to the field